prow/config/labels.yaml

```diff
@@ -1813,114 +1820,156 @@ repos:
         isExternalPlugin: true
         addedBy: anyone
-      - name: sig/raft
+      - name: area/bigdata
         color: "e99695"
-        description: Indicates that the Issue or PR belongs to the raft SIG.
+        description: Indicates that the Issue or PR belongs to the area of TiFlash, TiSpark, and OLAP.
         target: both
         prowPlugin: ti-community-label
         isExternalPlugin: true
         addedBy: anyone
-      - name: sig/scheduling
+      - name: area/planner
         color: "e99695"
-        description: Indicates that the Issue or PR belongs to the scheduling SIG.
+        description: Indicates that the Issue or PR belongs to the area of SQL planner or optimizer.
         target: both
         prowPlugin: ti-community-label
         isExternalPlugin: true
         addedBy: anyone
-      - name: sig/docs
+      - name: area/sql-infra
         color: "e99695"
-        description: Indicates that the Issue or PR belongs to the docs SIG.
+        description: Indicates that the Issue or PR belongs to the area of sql-infra and sql-metadata.
         target: both
         prowPlugin: ti-community-label
         isExternalPlugin: true
         addedBy: anyone
-      - name: sig/k8s
+      - name: area/transaction
         color: "e99695"
-        description: Indicates that the Issue or PR belongs to the k8s SIG.
+        description: Indicates that the Issue or PR belongs to the area of transaction.
         target: both
         prowPlugin: ti-community-label
         isExternalPlugin: true
         addedBy: anyone
-      - name: sig/tiup
+      - name: area/coprocessor
         color: "e99695"
-        description: Indicates that the Issue or PR belongs to the tiup SIG.
+        description: Indicates that the Issue or PR belongs to the area of coprocessor.
         target: both
         prowPlugin: ti-community-label
         isExternalPlugin: true
         addedBy: anyone
-      - name: sig/community-infra
+      - name: area/engine
         color: "e99695"
-        description: Indicates that the Issue or PR belongs to the community-infra SIG.
+        description: Indicates that the Issue or PR belongs to the area of TP storage or Cloud storage.
         target: both
         prowPlugin: ti-community-label
         isExternalPlugin: true
         addedBy: anyone
-      - name: sig/web
+      - name: area/raft
         color: "e99695"
-        description: Indicates that the Issue or PR belongs to the web SIG.
+        description: Indicates that the Issue or PR belongs to the area of Raft.
+        target: both
+        prowPlugin: ti-community-label
+        isExternalPlugin: true
+        addedBy: anyone
+      - name: area/scheduling
+        color: "e99695"
+        description: Indicates that the Issue or PR belongs to the area of scheduling.
+        target: both
+        prowPlugin: ti-community-label
+        isExternalPlugin: true
+        addedBy: anyone
+      - name: area/docs
```

Comment:
I'm a little confused about this area, because it seems that almost all PR may be related to documentation.

Reply:
Yes, I think so.